woocommerce/woocommerce-ios
# Fix Crash in `BottomSheetViewController`

Fixes: #10250 

## Description
I encountered a crash on iOS 16.4, simulator iPhone 14 Pro Max, while I was trying to add a new product. In the end, I discovered that this crash happens in every screen where we use the `BottomSheetViewController` component from the WordPressUI library.

## Testing instructions
1. Tap add new physical product
2. Tap "Add more details"
3. Tap the "top line" in the bottom sheet.
4. Crash

Run this PR, you should not be able to expact a crash anymore. 

## Before merging to trunk
- [x] Point WordPressUI pod to the release 1.13.1


---
